src/components/timeline/timeline.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, Input, ViewEncapsulation } from '@angular/core';
+
+@Component({
+    selector: 'ux-timeline',
+    templateUrl: './timeline.component.html',
+    encapsulation: ViewEncapsulation.None
+})
+export class TimelineComponent {
+
+    constructor () {}
+
+    private _events: any[] = [];
+  
+    @Input('events')

```

Comment:
We agreed to implement `ux-timeline-event` as content children of the `ux-timeline` component. We made no mention of an `events` property, and the documentation states:

"Events are created with the <code>ux-timeline-event</code> component, which includes a customizable badge and informational panel."

So please adhere to the design and remove this input. It will be built as follows:

````html
<ux-timeline>
  <ux-timeline-event>...</ux-timeline-event>
  <ux-timeline-event>...</ux-timeline-event>
  <ux-timeline-event>...</ux-timeline-event>
</ux-timeline-event>
````

Reply:
Events property removed.